core/deployment/src/main/java/io/quarkus/deployment/steps/SubstrateAutoFeatureStep.java

```diff
@@ -29,7 +34,17 @@
 
 public class SubstrateAutoFeatureStep {
 
-    private static final String GRAAL_AUTOFEATURE = "io.quarkus/runner/AutoFeature";
+    private static final String GRAAL_AUTOFEATURE = "io/quarkus/runner/AutoFeature";
```

Comment:
Oh WOW, we're replacing a partly binary class name. Can we introduce a convension for constant class names that lets us know if it's a class name, or a class binary name ('.' or '/')?

Reply:
I know, I'm just saying we should once and for all decide on a suffix/terminology for these constants.  I've the same problem in my extensions.